applications/asset_tracker_v2/doc/ui_module.rst

```diff
@@ -0,0 +1,107 @@
+
+.. _led_indication:
+
+LED indication
+==============
+
+The module supports multiple LED patterns to visualize the operating state of the application.
+The following table describes the supported LED states:
+
++---------------------------+-------------------------+-----------------------+
+| State                     | Thingy:91 RGB LED       | nRF9160 DK solid LEDs |
++===========================+=========================+=======================+
+| LTE connection search     | Yellow, blinking        | LED1 blinking         |
++---------------------------+-------------------------+-----------------------+
+| GNSS fix search           | Purple, blinking        | LED2 blinking         |
++---------------------------+-------------------------+-----------------------+
+| Publishing data           | Green, blinking         | LED3 blinking         |
++---------------------------+-------------------------+-----------------------+
+| Active mode               | Light blue, blinking    | LED4 blinking         |
++---------------------------+-------------------------+-----------------------+
+| Passive mode              | Dark blue, slow blinking| LED4 slow blinking    |
++---------------------------+-------------------------+-----------------------+
+| Error                     | Red, static             | all 4 LEDs blinking   |
++---------------------------+-------------------------+-----------------------+
+| Completion of FOTA update | White, rapid blinking   | all 4 LEDs on         |
++---------------------------+-------------------------+-----------------------+
+
+Configuration options
+*********************
+
+.. _CONFIG_UI_THREAD_STACK_SIZE:
+
+CONFIG_UI_THREAD_STACK_SIZE - UI module thread stack size
+   This option configures the UI module's internal thread stack size.
+
+Module states
+*************
+
+The UI module continuously updates its internal state based on the mode and operating state of the application.
+This ensures that the correct LED pattern is displayed when the application changes its state of operation.
+The UI module has an internal state machine with the following states:
+
+* ``STATE_ACTIVE`` - The application is in the active mode, revert to the active mode LED pattern.
+* ``STATE_PASSIVE`` - The application is in the passive mode, revert to the passive mode LED pattern.
+
+   * ``SUB_STATE_GNSS_ACTIVE`` - The application is performing a GNSS search, revert to the GNSS search LED pattern.
+   * ``SUB_STATE_GNSS_INACTIVE`` - A GNSS search is not being performed.
+
+* ``STATE_SHUTDOWN`` - The module has been shut down after receiving a request from the utility module.
+
+
+Module events
+*************
+
+The :file:`asset_tracker_v2/src/events/ui_module_event.h` header file contains a list of various events sent by the module.
+
+Dependencies
+************
+
+This module uses the following |NCS| libraries and drivers:
+
+* :ref:`:ref:`dk_buttons_and_leds_readme`
```

Comment:
```suggestion
* :ref:`dk_buttons_and_leds_readme`
```
Yet another whoops. I missed the extra `:ref:` and of course the doc build failed because of it.